robot.py

```diff
@@ -120,6 +121,14 @@ def createObjects(self) -> None:
             self.vision_rot = Rotation3d.fromDegrees(0, -20.0, 0)

```

Comment:
I think this will crash on the test robot as is. Could we get some reasonable values here, so anyone using it to test other mechanisms (and not relying on localisation) don't have to debug this?

Reply:
Done